plugins/modules/net_tools/nmcli.py

```diff
@@ -210,7 +210,7 @@
         default: 300
     mac:
         description:
-            - This is only used with bridge - MAC address of the bridge.
+            - MAC address of the connection.
             - Note this requires a recent kernel feature, originally introduced in 3.15 upstream kernel.
```

Comment:
Maybe add something like this, so that people can't complain to us when it won't work for every device:
```suggestion
            - Note this requires a recent kernel feature, originally introduced in 3.15 upstream kernel.
              It also might not be supported by every network device.
```

Reply:
I would prefer to avoid this since it should never really happen. NetworkManager's own documentation doesn't have anything like that: https://developer.gnome.org/NetworkManager/stable/settings-802-3-ethernet.html

Reply:
I don't think this could ever happen too, but adding a line specifying "it might not be supported by every network device" won't do any harm, will it?

Reply:
I would prefer to keep the documentation clear and accurate and avoid any conjecture or speculation.